### DISCUSSION TITLE
Do the contribution

### DISCUSSION THREAD

**Discussion by ghost** [FCP-INDI/C-PAC#2093] [General]

**Comment by shnizzedy** [comment]
:wave: Hi @DharahasGoud,

C-PAC is a public project (I think you'd see <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories#about-repository-visibility"><img alt="Octobi Wan Catnobi" src="https://octodex.github.com/images/octobiwan.jpg" width="48px"></a> instead of the project at https://github.com/FCP-INDI/C-PAC if the project were private), and we welcome contributions.

Here are some resources to help you get started:
- [FCP-INDI/C-PAC/CONTRIBUTING](https://github.com/FCP-INDI/C-PAC/blob/develop/CONTRIBUTING.md)
- [Contributor Covenant Code of Conduct](https://github.com/FCP-INDI/C-PAC?tab=coc-ov-file#readme)
- [FCP-INDI/CONTRIBUTING](https://github.com/FCP-INDI/.github/blob/main/CONTRIBUTING.md)
- [Developer Documentation](https://fcp-indi.github.io/docs/nightly/developer/)
- [FCP-INDI/fcp-indi.github.io/CONTRIBUTING](https://github.com/FCP-INDI/fcp-indi.github.io/blob/source/CONTRIBUTING.md)
- [FCP-INDI/C-PAC_tutorials/CONTRIBUTING](https://github.com/FCP-INDI/C-PAC_tutorials/blob/main/CONTRIBUTING.md)

In case you don't already have a particular contribution in mind, the ["good first issue" tag](https://github.com/search?q=org%3AFCP-INDI%20is%3Aopen%20is%3Aissue%20label%3A%22good%20first%20issue%22%20&type=issues) is a good place to look for issues we think have a good scope for a first contribution.

**Comment by ghost** [comment]
Can you suggest some of the private Project to contribute to the project ?

**Reply by shnizzedy** [reply]
I'm sorry, I don't understand this question

**Comment by ghost** [comment]
I was saying that I want to contribute to the private respiratory of the
university ?

**Reply by shnizzedy** [reply]
Ah, in that case, I think <span title="this is not the web page you are looking for."><img width="859" alt="this is not the web page you are looking for." src="https://github.com/FCP-INDI/C-PAC/assets/5974438/8d24e8be-a685-467d-be5e-a792a5b327e1"></span>

For private repositories, if you're not an owner or collaborator, you shouldn't even be able to see if they exist. The FCP-INDI project is a collaborative project specifically about open science and as such, is neither a university nor do we have any active private repositories.

---

You can make your own private repositories. By their nature, no one that isn't part of a private repository can see contributions to that repository anyway.

**Comment by ghost** [comment]
Yes I can but I want to join the respiratory of the university ?

**Reply by shnizzedy** [reply]
FCP-INDI isn't a university, and all the members of the organization are [current or former employees of the Child Mind Institute](https://childmind.org/science/global-open-science/data-sharing-initiatives/). You're [welcome to contribute](https://github.com/FCP-INDI/C-PAC/discussions/2093#discussioncomment-9072811) without being a member.

**Comment by ghost** [comment]
Is this project is a part of any university ?

**Reply by shnizzedy** [reply]
Nope!
